UnityProject/Assets/Scripts/Chemistry/ReactionLib.cs

```diff
@@ -53,7 +44,8 @@ public static void run()
             }
             private static void JsonImportInitialization()
             {
-                string json = File.ReadAllText(@"Reactions.json");
+			var path = Path.Combine(Environment.CurrentDirectory, @"Assets\Resources\Metadata\Reactions.json");
```

Comment:
This path won't exist in a build. Use Resources.Load instead: https://docs.unity3d.com/ScriptReference/Resources.Load.html

So it will be `var json = Resources.Load<TextAsset>("MetaData/Reactions")`;